src/Microsoft.DotNet.Darc/DarcLib/Helpers/ManifestHelper.cs

```diff
@@ -86,19 +121,39 @@ List<string> GetTargetPaths(DownloadedAsset asset)
         {
             if (makeAssetsRelativePaths)
             {
-                return new List<string>
+                var list = new List<string>()
                 {
-                    Path.GetRelativePath(outputPath, asset.ReleaseLayoutTargetLocation),
                     Path.GetRelativePath(outputPath, asset.UnifiedLayoutTargetLocation)
                 };
+
+                if (asset.SeparatedLayoutTargetLocation != null)
+                {
+                    list.Add(Path.GetRelativePath(outputPath, asset.SeparatedLayoutTargetLocation));
+                }
+
+                if (asset.ReleasePackageLayoutTargetLocation != null)
+                {
+                    list.Add(Path.GetRelativePath(outputPath, asset.ReleasePackageLayoutTargetLocation));
+                }
+                return list;
             }
             else
             {
-                return new List<string>
+                var list = new List<string>()
                 {
-                    asset.ReleaseLayoutTargetLocation,
                     asset.UnifiedLayoutTargetLocation
                 };
+
+                if (asset.SeparatedLayoutTargetLocation != null)
+                {
+                    list.Add(asset.SeparatedLayoutTargetLocation);
+                }
+
+                if (asset.ReleasePackageLayoutTargetLocation != null)
+                {
+                    list.Add(asset.ReleasePackageLayoutTargetLocation);
+                }
+                return list;
             }
```

Comment:
I think this block can be simplified to this, with the introduction of the helper method below:

```csharp
                var list = new List<string>();
                list.Add(GetPath(asset.UnifiedLayoutTargetLocation, outputPath, makeAssetsRelativePaths));

                if (asset.SeparatedLayoutTargetLocation != null)
                {
                    list.Add(GetPath(asset.SeparatedLayoutTargetLocation, outputPath, makeAssetsRelativePaths));
                }

                if (asset.ReleasePackageLayoutTargetLocation != null)
                {
                    list.Add(GetPath(asset.ReleasePackageLayoutTargetLocation , outputPath, makeAssetsRelativePaths));
                }
                return list;

                private static string GetPath(string path, string outputPath, bool makeAssetsRelativePaths) => makeAssetsRelativePaths ? path : Path.GetRelativePath(outputPath, path);

```